### ISSUE TITLE
Do you have any plan to update this package?

### ISSUE THREAD

**Issue by Hanggi** [pay-2-bit-human/dart_web3#1]
Maybe we could open discussion?

**Comment by anipy1** [comment]
Yes, Well if you also want to implement a feature in dart_web3, you can.

**Comment by MuTe33** [comment]
the official web3dart package was being handed over and continued. We should aim to continue there as many people who are using that package will have a look there first. People do not need to touch their dependency. Also as you can see some contribution started happening again. Therefore there is no need for this. if you don't plan to go a different route of course :)

https://pub.dev/packages/web3dart

**Comment by anipy1** [comment]
Ownership of the web3dart package is transferred, that's really great! Well, I've also been using web3dart in my personal projects for a long time. The point of dart_web3 is if there is any problem for other developers in the library then anyone can solve it because web3dart is EOL and the repository is read only. Now that the ownership is transferred, it's entirely up to each developer which package they want to use. I'm staying with dart_web3 and developing a bit of dapp mobile wallet  with it.
